### ISSUE TITLE
tinySSD training loss (mbox_loss) is not decreasing below 8.0

### ISSUE THREAD

**Issue by vasoyajignesh7** [lampsonSong/tinySSD#14]
Hi,

I am trying to train model for COCO 2017 dataset. mbox_loss reduces up to 8.0 within 1k iterations. But loss is not decreasing less than 8.0 even after more than 1 lac iteration run. I have also tried different hyper parameter values like batch size, base_lr, gamma, lr_policy (mutistep, Poly) and optimization (RMSprop, sgd) but not success yet.

Great if you can suggest suitable solution to achieve best training loss/ mbox_loss.
For your reference attaching training log file and plot which is achieved best so far after multiple trials.

Hope you will be able to provide solution soon.
[coco_train_log_Trial_2.log](https://github.com/lampsonSong/tinySSD/files/4856336/coco_train_log_Trial_2.log)
![train_Trial_2](https://user-images.githubusercontent.com/20473126/86216180-3ca51c00-bb9b-11ea-98aa-2c40c995fb06.png)
